sessionctx/binloginfo/binloginfo.go

```diff
@@ -55,6 +58,40 @@ func GetPumpsClient() *pumpcli.PumpsClient {
 	return client
 }
 
+// GetOrCreatePumpsClient get or create the pumps client instance
+func GetOrCreatePumpsClient() *pumpcli.PumpsClient {
+	pumpsClientLock.Lock()
+	client := GetPumpsClient()
+	if client == nil {
+		var err error
+
+		securityOption := pd.SecurityOption{
```

Comment:
some code is same with function `setupBinlogClient` in main.go, can we define a function to  do these things and then call it?

Reply:
Done.